radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt

```diff
@@ -136,9 +139,40 @@ class RadialGraph : ConstraintLayout {
         }
     }
 
+    fun setGraphNode(newGraphNodeType: GraphNode) {
+        graphConfig.graphNodeType = newGraphNodeType
+    }
+
+    fun setBackgroundTrack(newColor: Int?) {
+        graphConfig.backgroundTrackDrawable = null
+        graphConfig.backgroundTrackColor = newColor ?: NO_ID
+    }
+
+    fun setBackgroundTrack(newDrawable: Drawable?) {
+        graphConfig.backgroundTrackColor = NO_ID
+        graphConfig.backgroundTrackDrawable = newDrawable
+    }
+
+    fun removeBackgroundTrack() {
+        with (graphConfig) {
+            backgroundTrackColor = NO_ID
+            backgroundTrackDrawable = null
+        }
+    }
```

Comment:
Won't `setBackgroundTrack(null)` automatically do this? I feel this function may be unncessary.

Reply:
The problem here is that `setBackgroundTrack` has 2 signatures and cannot resolve when you pass `null`

Reply:
If you qualify either of the parameters:
`setBackgroundTrack(newDrawable = null)` or `setBackgroundTrack(newColor = null)`
It will work, so it's still workable, or do you disagree?

Reply:
While workable, it might not be intuitive that this is a possibility. That could be remedied by adding kdocs to the `setBackgroundTrack` function though, so I will settle on that route